README.md

```diff
@@ -1,15 +1,47 @@
 # Theory vs. Practice
+I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice.
+
+I talked to some kid in class about this and I don't know his name, we just bounced ideas off each other.
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. Asymptotic analysis ignores constants which doesn't give you the full
+     realistic performance in practice.
+  2. There is a difference between worst, best and average case. For an arbitrary
+     algorithm, the worst case may be $n^2$, but the average case is $n$. So in
+     practice, it might actually run faster or slower than expected using asymptotic
+     analysis.
```

Comment:
Not sure if I understand this reason -- are you saying that you might assume best case, but it turns out to be worst case? That would be a problem with the analysis, not with asymptotic complexity as such.